parachain/src/chain_spec.rs

```diff
@@ -275,6 +275,10 @@ pub fn staging_testnet_config(id: ParaId) -> TestnetChainSpec {
                     ]),
                 ],
                 vec![
+                    // 5Ec37KSdjSbGKoQN4evLXrZskjc7jxXYrowPHEtH2MzRC7mv (//sudo/1)
+                    get_account_id_from_string("5Ec37KSdjSbGKoQN4evLXrZskjc7jxXYrowPHEtH2MzRC7mv"),
+                    // 5ECj4iBBi3h8kYzhqLFmzVLafC64UpsXvK7H4ZZyXoVQJdJq (//oracle/1)
+                    get_account_id_from_string("5ECj4iBBi3h8kYzhqLFmzVLafC64UpsXvK7H4ZZyXoVQJdJq"),
```

Comment:
Should we remove the account/1-5 and add the vault?

Reply:
These are just random accounts we can use for the vaults

Reply:
Not sure what is most convenient, I'll leave it up to you